RetrospectiveExtension.Frontend/utilities/external/telemetryClient.ts

```diff
@@ -7,11 +7,26 @@ const appInsights = new ApplicationInsights({
   config: {
     instrumentationKey: environment.AppInsightsInstrumentKey,
     extensions: [reactPlugin],
+    loggingLevelConsole:2,
+    loggingLevelTelemetry:2,
     extensionConfig: {
       [reactPlugin.identifier]: { history: {} }
     }
   }
 });
 appInsights.loadAppInsights();
 
+// The new log error function logs the data to console as usual,
+// but also sends error messages to Application insights as well.
+const updatedConsoleError=(function(oldErrorFunction){
```

Comment:
Can we rewrite this function using arrow function syntax? Most of our methods are arrow functions since it would be great to be consistent with the rest.

Reply:
done